app/start.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+cd "${0%/*}" || exit 1
```

Comment:
Can you put a comment explaining this line, to help devs looking at the file?

Also, could you make a similar change to start.ps1? We need to keep them working similarly.

Reply:
the comment is added